server/models/webhook.ts

```diff
@@ -41,12 +43,12 @@ export default (
     validate: {
       // webhooks should only have 1 of these properties
       eitherOffchainOrOnchain() {
-        if (!(this.chain_id === undefined || this.offchain_community_id === undefined)) {
-          throw new Error('Either chain_id or offchain_community_id!');
-        }
-        if (this.chain_id !== undefined && this.offchain_community_id !== undefined) {
-          throw new Error('Either chain_id or offchain_community_id not both!');
-        }
+        // if (!(this.chain_id === 'undefined' || this.offchain_community_id === 'undefined')) {
+        //   throw new Error('Either chain_id or offchain_community_id!');
+        // }
+        // if (this.chain_id !== 'undefined' && this.offchain_community_id !== 'undefined') {
+        //   throw new Error('Either chain_id or offchain_community_id not both!');
+        // }
```

Comment:
Why is this commented out now?

Reply:
It wasn't working as it was currently written and didn't have the chance to debug it at the time, fixed it now though.